.gcloudignore

```diff
@@ -0,0 +1,2 @@
+#!include:.gitignore
```

Comment:
`.gitignore` includes `build`. Does this prevent the build folder from getting uploaded?

Reply:
Good call. Since the `.gitgnore` isn't present on the CI server, it isn't taken into consideration anyway, so I'll just remove this.